packages/bratus-cli/src/api/Server.ts

```diff
@@ -1,8 +1,8 @@
 import ASTParser from '@react-bratus/parser';
```

Comment:
This showed an error.
The Server.ts couldn't find ASTParser at this location.

Should this be updated to the new location?

`import ASTParser from '../parser/ASTParser';`

Reply:
Hi @thepiotrek,

Yes, you are correct.